### DISCUSSION TITLE
TreePlanterError

### DISCUSSION THREAD

**Discussion by AlexKDo** [UMEP-dev/UMEP#507] [Q&A] [answered]
Hello.

I am trying to run TreePlanter but I have some error. I attach the image.
![image](https://user-images.githubusercontent.com/81244161/225417023-9ca84202-8ac6-45e6-8046-36f5daa4eb6f.png)

**Comment by nilswallenberg** [comment]
Hi,

Looks to me like you have your end hour after your start hour.

Cheers,
Nils

**Reply by AlexKDo** [reply]
maybe... silly me 📦 But still.. I tries with the material from the tutorial and error again. 
![image](https://user-images.githubusercontent.com/81244161/225570956-b3456847-adf9-440b-ba7f-e61bfc601e38.png)

**Reply by AlexKDo** [reply]
and this is my met file 
![image](https://user-images.githubusercontent.com/81244161/225571414-f3cda4a6-52af-4c23-8a01-5970cdc9a66c.png)

**Comment by nilswallenberg** [comment, marked as answer]
What version of UMEP/SOLWEIG are you using? Did you run SOLWEIG in UMEP from the processing toolbox or the other version?

**Reply by AlexKDo** [reply]
i runned solweig from qgis toolbar and not from processing toolbox. So I will to run it from toobox and try tre planter again?

**Reply by AlexKDo** [reply]
umep version 3.20.9

**Reply by nilswallenberg** [reply]
Yes! Or download the latest version of UMEP for toolbar (here at github) and run again. Should work there too with the latest version. :)

**Reply by AlexKDo** [reply]
So quick question for installation. Should I uninstall my current version and reinstall umep again?

**Reply by nilswallenberg** [reply]
Yes, try that

**Reply by AlexKDo** [reply]
It worked thank you!

**Reply by nilswallenberg** [reply]
Great! Sorry for the inconvenience. :)
